crates/scroll/bin/scroll-reth/Cargo.toml

```diff
@@ -38,4 +38,5 @@ optimism = [
 
 [[bin]]
 name = "scroll-reth"
-path = "src/main.rs"
\ No newline at end of file
+path = "src/main.rs"
+required-features = ["scroll"]
```

Comment:
nit: line break

Reply:
This was added intentionally.

Reply:
you mean you meant to remove the line break?

Reply:
Oh sorry I thought you meant the introduction of the new line at eof. What do you mean about the line break?

Reply:
I mean you added a new line at the end of the file, we can merge as is for me, doesn't matter much